pkgs/os-specific/linux/ply/default.nix

```diff
@@ -7,7 +7,7 @@ let
 in stdenv.mkDerivation {
   pname = "ply";
   inherit version;
-  nativeBuildInputs = [ autoreconfHook flex yacc p7zip ];
+  nativeBuildInputs = [ autoreconfHook flex yacc p7zip rsync ];
```

Comment:
Why should rsync be added? Please say why in the commit message. (This package builds fine on master and release-19.09, so it cannot be a build fix.)

Reply:
It seems to be a problem in `linuxPackages_latest.ply` according to hydra: https://hydra.nixos.org/eval/1543648?filter=.ply&compare=1543612&full=#tabs-still-fail